desktop/src/com/unciv/app/desktop/MultiplayerTurnNotifierWindows.kt

```diff
@@ -9,10 +9,9 @@ import com.sun.jna.Pointer
 import com.sun.jna.platform.win32.User32
 import com.sun.jna.platform.win32.WinNT
 import com.sun.jna.platform.win32.WinUser
-import com.unciv.ui.MultiplayerTurnNotifier
 import org.lwjgl.glfw.GLFWNativeWin32
 
-class MultiplayerTurnNotifierWindows: Lwjgl3WindowAdapter(), MultiplayerTurnNotifier {
+class MultiplayerTurnNotifierWindows: Lwjgl3WindowAdapter() {
```

Comment:
I see. I had the vague idea the meat and bones of this was small enough to be folded into the helper - but yea, if it also needs to implement an API surface in another direction maybe better separate those.